releases/mainnet/1.9.12/index.html

```diff
@@ -0,0 +1,27 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.9.12</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.9.12</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.9.12/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmYojffjj3t9ifkkyATD9xr2zPMPsvpMgcoqzVtppAhNKQ">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.9.12/abi/index.html">ABIs</a></li>
+
+				<li>Links to GitHub repositories
```

Comment:
to check with @gbalabasquer about that, but I think we could keep it commented for now

Reply:
Leave it uncommented. I'll get `dss-deploy-scripts` 1.9.12 soon.
Btw I'm realizing I forgot to push 1.9.11, will do that soon as well.

Reply:
1.9.11 and 1.9.12 releases added to `dss-deploy-scripts`.